aspnetcore/fundamentals/error-handling.md

```diff
@@ -204,40 +226,49 @@ public class ErrorModel : PageModel
 }
 ```
 
+::: moniker-end
+
 ## Exception-handling code
 
 Code in exception handling pages can throw exceptions. It's often a good idea for production error pages to consist of purely static content.
 
-Also, be aware that once the headers for a response have been sent, you can't change the response's status code, nor can any exception pages or handlers run. The response must be completed or the connection aborted.
+Also, be aware that once the headers for a response are sent:
+
+* The app can't change the response's status code.
+* Any exception pages or handlers can't run. The response must be completed or the connection aborted.
 
 ## Server exception handling
 
-In addition to the exception handling logic in your app, the [server](xref:fundamentals/servers/index) hosting your app performs some exception handling. If the server catches an exception before the headers are sent, the server sends a *500 Internal Server Error* response with no body. If the server catches an exception after the headers have been sent, the server closes the connection. Requests that aren't handled by your app are handled by the server. Any exception that occurs is handled by the server's exception handling. Any configured custom error pages or exception handling middleware or filters don't affect this behavior.
+In addition to the exception handling logic in your app, the [server](xref:fundamentals/servers/index) hosting your app performs some exception handling. If the server catches an exception before the headers are sent, the server sends a *500 - Internal Server Error* response without a response body. If the server catches an exception after the headers are sent, the server closes the connection. Requests that aren't handled by your app are handled by the server. Any exception that occurs is handled by the server's exception handling. Any configured custom error pages or exception handling middleware or filters don't affect this behavior.
```

Comment:
"hosting your app" could be confused with the web host/generic host ("hosting layer" in the following paragraph); I suggest "the server implementation that your app uses" or something similar.